app/src/main/kotlin/com/adesso/movee/internal/injection/module/AppModule.kt

```diff
@@ -2,23 +2,19 @@ package com.adesso.movee.internal.injection.module
 
 import android.app.Application
 import android.content.Context
-import com.adesso.movee.internal.injection.DaggerApplication
 import dagger.Module
 import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
 import javax.inject.Singleton
 
 @Module(includes = [MoshiModule::class])
-internal class AppModule {
+@InstallIn(SingletonComponent::class)
+object AppModule {
 
     @Provides
     @Singleton
-    internal fun provideApplicationContext(application: DaggerApplication): Context {
+    fun provideApplicationContext(application: Application): Context {
```

Comment:
Hilt already provides application context with `@ApplicationContext` annotation, this function can be removed.

Reply:
Thank you for review. Totally agreed. Will fix it asap.

Reply:
I removed that related codes, can you check it out?